x/superfluid/keeper/distribution_test.go

```diff
@@ -55,6 +55,14 @@ func (suite *KeeperTestSuite) TestMoveSuperfluidDelegationRewardToGauges() {
 			[]int64{0},
 			[]gaugeChecker{{0, 0, 0, true}, {1, 1, 0, false}},
 		},
+		{
```

Comment:
can we add more detailed comments for what we're expecting from this test case?
e.g) unbonding validators are not expected to be rewarded

Reply:
It was actually not allocating the reward to the Unbonded validator. Therefore gauges are not distributed.
I added the case of allocating to both Unbonded and Unbonding vals and gauges are distributed normally.
Also in these tests, except for the case that slash does not work with Unbonded validator. I think there is no difference between the 3 stats together